src/libraries/System.Net.Primitives/src/System/Net/IPAddress.cs

```diff
@@ -633,15 +630,7 @@ public bool IsIPv4MappedToIPv6
         {
             get
             {
-                if (IsIPv4)
-                {
-                    return false;
-                }
-
-                ReadOnlySpan<byte> numbers = MemoryMarshal.AsBytes(new ReadOnlySpan<ushort>(_numbers));
-                return
-                    MemoryMarshal.Read<ulong>(numbers) == 0 &&
-                    BinaryPrimitives.ReadUInt32LittleEndian(numbers.Slice(8)) == 0xFFFF0000;
+                return !IsIPv4 && _numbers.AsSpan(0, 6).SequenceEqual((ReadOnlySpan<ushort>)[0, 0, 0, 0, 0, 0xFFFF]);
```

Comment:
Instead of:
```C#
SequenceEqual((ReadOnlySpan<ushort>)[0, 0, 0, 0, 0, 0xFFFF])
```
could it be:
```C#
SequenceEqual<ushort>([0, 0, 0, 0, 0, 0xFFFF])
```
?

Reply:
good point!